playbooks/aws/openshift-cluster/vars.yml

```diff
@@ -31,3 +31,100 @@ deployment_vars:
   enterprise: "{{ deployment_rhel7_ent_base }}"
   openshift-enterprise: "{{ deployment_rhel7_ent_base }}"
   atomic-enterprise: "{{ deployment_rhel7_ent_base }}"
+
+provision:
```

Comment:
As noted before, setting variables in include_vars files makes them difficult to override for users.  You'll notice, most of the other variables set in this file refer back to other variables.